### PR TITLE
Code format and comment change in dispatcher; no code logic change involved.

### PR DESCRIPTION
**Pull request by eurekaka**

### CODE REVIEW

**Comment by xiong-gang** [comment]
LGTM

File: src/backend/cdb/dispatcher/cdbdisp_dtx.c [hunk 1]
```diff
@@ -373,7 +373,7 @@ PQbuildGpDtxProtocolCommand(MemoryContext cxt,
 
 	*pos++ = 'T';
 
-	pos += 4;					/* place holder for message length */
+	pos += 4; /* place holder for message length */
```
Comment (danielgustafsson) [inline src/backend/cdb/dispatcher/cdbdisp_dtx.c]: s/place holder/placeholder/

File: src/backend/cdb/dispatcher/cdbdisp_query.c [hunk 2]
```diff
@@ -1017,18 +1013,7 @@ fillSliceVector(SliceTable *sliceTbl, int rootIdx,
...
- * This should *ONLY* ever be used by the Greenplum Database Query Dispatcher and NEVER (EVER)
- * by anyone else.
- *
- * snapshot - serialized form of Snapshot data.
- * xid		- Global Transaction Id to use.
- * flags	- specifies additional processing instructions to the remote server.
- *			  e.g.	Auto explicitly start a transaction before executing this
- *			  statement.
- * gp_command_count - Client request serial# to be passed to the qExec.
+ * Build a query string to be dispatched bo QE.
```
Comment (danielgustafsson) [inline src/backend/cdb/dispatcher/cdbdisp_query.c]: s/bo/to/

File: src/backend/cdb/dispatcher/cdbdisp_query.c [hunk 3]
```diff
@@ -1099,7 +1084,7 @@ PQbuildGpQueryString(MemoryContext cxt, DispatchCommandParms * pParms,
 
 	*pos++ = 'M';
 
-	pos += 4;					/* place holder for message length */
+	pos += 4; /* place holder for message length */
```
Comment (danielgustafsson) [inline src/backend/cdb/dispatcher/cdbdisp_query.c]: s/place holder/placeholder/

File: src/backend/cdb/dispatcher/cdbdisp_query.c [hunk 4]
```diff
@@ -1331,17 +1316,6 @@ cdbdisp_dispatchX(DispatchCommandQueryParms *pQueryParms,
...
-	 * Now we need to call CDBDispatchCommand once per slice.  Each such
-	 * call dispatches a MPPEXEC command to each of the QEs assigned to
-	 * the slice.
-	 *
-	 * The QE information goes in two places: (1) in the argument to the
-	 * function CDBDispatchCommand, and (2) in the serialized
-	 * command sent to the QEs.
-	 *
-	 * So, for each slice in the tree...
-	 */
```
Comment (danielgustafsson) [inline src/backend/cdb/dispatcher/cdbdisp_query.c]: This comment conveys meaning, if it's incorrect I'd rather see it reworded to indicate the purpose of the for loop.

File: src/include/cdb/cdbgang.h [hunk 5]
```diff
@@ -13,60 +13,57 @@
...
-	GANGTYPE_UNALLOCATED,       /* a root slice executed by the qDisp */
-	GANGTYPE_ENTRYDB_READER,    /* a 1-gang with read access to the entry db */
-	GANGTYPE_SINGLETON_READER,  /* a 1-gang to read the segment dbs */
-	GANGTYPE_PRIMARY_READER,    /* a N-gang to read the segment dbs */
-	GANGTYPE_PRIMARY_WRITER     /* the N-gang that can update the segment dbs */
+	GANGTYPE_UNALLOCATED, /* a root slice executed by the qDisp */
+	GANGTYPE_ENTRYDB_READER, /* a 1-gang with read access to the entry db */
+	GANGTYPE_SINGLETON_READER, /* a 1-gang to read the segment dbs */
+	GANGTYPE_PRIMARY_READER, /* a N-gang to read the segment dbs */
+	GANGTYPE_PRIMARY_WRITER /* the N-gang that can update the segment dbs */
```
Comment (danielgustafsson) [inline src/include/cdb/cdbgang.h]: Lining up the member comments is what most postgres header files do, it seems reasonable to stick that convention.

File: src/include/cdb/cdbgang.h [hunk 6]
```diff
@@ -13,60 +13,57 @@
...
 	 */
 	struct SegmentDatabaseDescriptor *db_descriptors;	
 
 	/* For debugging purposes only. These do not add any actual functionality. */
-	bool		allocated;
+	bool allocated;
 
-	/* should be destroyed in cleanupGang() if set*/
-	bool		noReuse;
+	/* should be destroyed if set*/
```
Comment (danielgustafsson) [inline src/include/cdb/cdbgang.h]: Add whitespace between last word and end-of-comment marker.

**Comment by eurekaka** [comment]
Merged with comments resolved, thanks for reviewing.

**Pull request closed without merging**